metashare/api/models.py

```diff
@@ -132,6 +132,22 @@ def __str__(self):
         return self.url
 
 
+class ProjectSlug(AbstractSlug):
+    parent = models.ForeignKey(
+        "Project", on_delete=models.PROTECT, related_name="slugs"
+    )
+
+
+class Project(mixins.HashIdMixin, mixins.TimestampsMixin, SlugMixin, models.Model):
```

Comment:
@wlonk I don't see anywhere that a `project` is owned by a `product` -- I think we need a relation here?

Reply:
Yeah, not sure how I missed that besides general high levels of stress and ball-dropping last week :(

Reply:
Eugh, I added it and made a migration and everything, what happened. I'll fix it tonight.

Reply:
@jgerigmeyer fixed.